mesonbuild/interpreterbase/interpreterbase.py

```diff
@@ -183,7 +183,6 @@ def evaluate_codeblock(self, node: mparser.CodeBlockNode, start: int = 0, end: T
         while i < len(statements):
             cur = statements[i]
             try:
-                self.current_lineno = cur.lineno
```

Comment:
I was somehow wondering if this was used for cases where we might evaluate something internally and want more accurate tracking for an outer loop, but nope, it literally isn't used anywhere meaningful. :/